client/src/pages/TakeSurvey/SurveyDescription/SurveyDescription.jsx

```diff
@@ -26,6 +44,8 @@ const SurveyDescription = () => {
       <Typography data-testid='survey-disclaimer' variant='h5'>
         {survey.disclaimer}
       </Typography>
+      {survey.anonymous === true && <AnonymousDisclaimer />}
```

Comment:
ternary would be ok here

Reply:
how about `survey.anonymous && <AnonymousDisclaimer/>`?